### PR TITLE
#634 Enforce at-clause order

### PR DESCRIPTION
**Pull request by mkordas**
For #634:
* specify order of at-clauses (Oracle recommended order, then `@todo`, then `@checkstyle`
* add test

### CODE REVIEW

**Comment by mkordas** [comment]
@davvd please assign reviewer

**Comment by davvd** [comment]
@mkordas Thanks, I will find someone to review this PR soon

**Comment by davvd** [comment]
> @davvd please assign reviewer

@mkordas OK, got it

**Comment by davvd** [comment]
@dmzaytsev review this one, please

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by krzyk** [comment]
@rultor merge pls

File: .rultor.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ assets:
   s3cfg: yegor256/home#assets/s3cfg
 merge:
   script: |-
-    pdd --source=$(pwd) --verbose --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java
+    pdd --source=$(pwd) --verbose --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java,qulice-checkstyle/src/test/resources/com/qulice/checkstyle/AtClauseOrder.java
```
Comment (dmzaytsev) [inline .rultor.yml]: @mkordas did you test the command line locally?

**Comment by dmzaytsev** [comment]
@krzyk I think we need to merge `rultor.yml` first

File: .rultor.yml (hunk 1 above)
Comment (dmzaytsev) [inline .rultor.yml]: @mkordas ignore my previous comment

**Comment by mkordas** [comment]
@krzyk please merge #645 first

**Comment by mkordas** [comment]
@krzyk can we try merging this one now?

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by mkordas** [comment]
@krzyk @dmzaytsev isn't it bug/serious limitation of Rultor?

**Comment by mkordas** [comment]
@rultor deploy

**Comment by krzyk** [comment]
@rultor deploy

**Comment by krzyk** [comment]
@mkordas not at all, see https://github.com/yegor256/rultor/issues/802#issuecomment-113381625

**Comment by mkordas** [comment]
@krzyk interesting, good to know

**Comment by davvd** [comment]
@dmzaytsev Thanks so much! Your account was topped up for **19 mins** (transaction ID is `75788481`, the task took 24 hours and 38 mins)... you're getting extra minutes here (c=4)... added +19 to your rating, now it is equal to [+2775](http://www.netbout.com/b/35166?open=rating)

**Comment by davvd** [comment]
@rultor deploy pls

**Pull request merged**